src/index.bundlespec.js

```diff
@@ -9,6 +9,12 @@ describe('exports', () => {
     expect(customProperties).toBeTruthy();
   });
 
+  describe('version', () => {
+    it('should match package.json version', () => {
+      expect(version).toMatch(process.env.npm_package_version);
```

Comment:
bundle tests 😄

Reply:
or should I directly import from package.json 🤔

Reply:
I think either is fine.